front/src/styles.js

```diff
@@ -0,0 +1,29 @@
+import SRRButton from "./SRRButton";
+import styled from "styled-components";
+
+export const AppWrap = styled.div`
+  height: 100vh;
+`;
+
+export const BottomBar = styled.footer`
+  position: fixed;
+  background-color: white;
+  bottom: 0;
+  width: 100%;
+  height: 10%;
+  border-style: solid;
+  border-width: 2px 0px 0px 0px;
+  display: flex;
+  justify-content: space-between;
+  margin: 0px;
+`;
+
+export const DefaultButton = styled(SRRButton)`
+  background: #ededed;
+  width: 32%;
+  font-size: 5vmin;
+  flex: 1;
+  &:active {
+    background: silver;
+  }
+`;
```

Comment:
Why not have the styles in the specific component file? This is just a normal stylesheet, which is what SC was trying to avoid